ql/src/test/org/apache/hadoop/hive/ql/TestTxnCommands3.java

```diff
@@ -52,6 +55,7 @@
 
 import static org.apache.hadoop.hive.ql.lockmgr.TestDbTxnManager2.swapTxnManager;
 import static org.mockito.Matchers.any;
+import static org.powermock.api.mockito.PowerMockito.when;
```

Comment:
Can the usage of `powermock` be avoided?

Reply:
Yes, accidentally added wrong import